lib/workers/repository/update/pr/body/config-description.ts

```diff
@@ -62,3 +54,25 @@ export async function getPrConfigDescription(
   }
   return prBody;
 }
+
+function scheduleToString(
+  schedule: string[] | undefined,
+  timezone: string | undefined
+): string {
+  let scheduleString = '';
+  if (
+    schedule &&
+    (schedule as never) !== 'at any time' &&
```

Comment:
@rarkins  I don't think wee need this check, it should always be a string array here. 🤔 
```suggestion
```

Reply:
Are your types strict enough to be sure of that?

Reply:
i think our migration already handles this to array, so this should only happen because of wrong tests.

Reply:
this is confirmed in the logs and in debug mode as well

```
DEBUG: Found renovate.json config file (repository=ladzaretti/automerge-schedule)
DEBUG: Repository config (repository=ladzaretti/automerge-schedule)
      ...
             "automergeSchedule": "before 2pm",
             "schedule": "at any time"
      ...
DEBUG: migrateAndValidate() (repository=ladzaretti/automerge-schedule)
DEBUG: No config migration necessary (repository=ladzaretti/automerge-schedule)
DEBUG: massaged config (repository=ladzaretti/automerge-schedule)
      ...
             "automergeSchedule": ["before 2pm"],
             "schedule": ["at any time"]
      ...
```

Reply:
and what about this?
https://github.com/renovatebot/renovate/blob/cf37fc1449ca96b54b57f0c91045b65f00e33c32/lib/workers/repository/update/branch/schedule.ts#L123-L127